code/modules/mob/mob.dm

```diff
@@ -1047,11 +1047,12 @@ Use this proc preferably at the end of an equipment loadout
 	if(istype(loc,/obj/mecha))
 		return
 
-	if(isVentCrawling())
+	var/obj/item/W = get_held_item_by_index(active_hand)
+
+	if(isVentCrawling() && !W.vent_use)
```

Comment:
I believe this runtimes if W does not exist (null.vent_use). I recommend moving this down to after the if(W) check/change that to a if(isnull(W)) return or such.